server/services/stats/side/tc_client.go

```diff
@@ -0,0 +1,52 @@
+package stats_side
+
+import (
+	"github.com/TheQuestionru/thequestion/server/schema"
+	"github.com/ivankorobkov/di"
+	"github.com/kapitanov/go-teamcity"
+)
+
+const (
+	buildsSize int = 40
+)
+
+func TcClientModule(m *di.Module) {
+	m.AddConstructor(NewTcClient)
+	m.MarkPackageDep(Config{})
+}
+
+type TcClient interface {
+	TcGetBuilds() ([]*schema.TcBuild, error)
+}
+
+type tcClient struct {
+	teamcitySvc teamcity.Client
+}
+
+func NewTcClient(config Config) TcClient {
+	if config.TcUrl == "http://localhost:8111" {
```

Comment:
i thought `"test"` is a magic string, but this is more 

if you try to access local TeamCity during tests - you shouldn't for a couple of reasons: this brings extra dependency, forces you (or your RE) to build it for tests, and gives same result as just mock